builder.md

```diff
@@ -70,15 +56,19 @@ Scheme files (excluding `base16`) have the following structure:
       base0D: "#dddddd"
       base0E: "#eeeeee"
       base0F: "#ffffff"
+    mappings:
+      diff_added_bg: base0B
 
```

Comment:
```suggestion
    mappings:
      diff_added_bg: base0F
      punctuation: ui_background_brightest

```

I think it needs to be clear these can resolve to other mappings/slots.   I also think `diff_added_bg: base0B` is a really bad **anti-example** and as that's _one of the big problems we're trying to fix_ with semantics - _the idea that base0B is a given hue_. This is why mappings make far more sense pointing to variables and other mappings than they do to the existing base schema.

Reply:
I'm dropping mappings from here for now, since that will be a part of base-next and these changes are closer to what I had in mind for base16v2.